lidar/README.md

```diff
@@ -0,0 +1,56 @@
+CEDAR-lidar
+---
+# Build package in docker
+
+1. Install Docker: https://docs.docker.com/get-docker/
+2. Install Docker compose: https://docs.docker.com/compose/install/ (for server)
+
+### Build docker image
+```bash
+cd lidar # Go to where Dockerfile is 
+docker build --tag lidar . # build the image
+ docker run --rm -it --entrypoint bash lidar 
+```
+# Build package locally
+
+### Install poetry
+1. install Python poetry: https://python-poetry.org/docs/#installation
+2. make sure your local python version is `3.8.6`. I recommand use `pyenv` to set up a local python version.
```

Comment:
shouldn't this be part of the docker?

Reply:
yeah that's true, maybe u can help me decide:
1. use poetry to install everything, even in docker
2. use poetry to export a requirements.txt, and then use that to install
which way should we use?

Reply:
seems easiest to do everything in docker so the user doesn't have to install anything locally on their machine?